adapters/eplanning/eplanning.go

```diff
@@ -26,6 +27,10 @@ const sec = "ROS"
 const dfpClientID = "1"
 const requestTargetInventory = "1"
 
+var isMobile = -1
```

Comment:
Adapters must not modify package scope variables. Due to the highly concurrent nature of PBS, this will most certainly result in race conditions and panics. To solve this, I recommend splitting the sorter into two entites such as `byPriorityMobile` and `byPriorityDesktop` created by passing down the `isMobile` flag.